app/pages/about/about-home.spec.js

```diff
@@ -6,17 +6,12 @@ import { shallow } from 'enzyme';
 import AboutHome from './about-home';
 
 describe('AboutHome', function () {
-  let wrapper;
-
-  before(function () {
-    wrapper = shallow(<AboutHome />);
-  });
-
   it('renders without crashing', function () {
```

Comment:
similar to https://github.com/zooniverse/Panoptes-Front-End/pull/4224/files#diff-68a4f0f6c9db644e47eab002cca11ca1R9